internal/tool/generate/generator.go

```diff
@@ -548,7 +548,7 @@ type component struct {
 }
 
 func fullName(t *types.Named) string {
-	return filepath.Join(t.Obj().Pkg().Path(), t.Obj().Name())
```

Comment:
Does `path.Join(t.Obj().Pkg().Path(), t.Obj().Name())` work? I replaced the `filepath` package with the `path` package.

Reply:
Yes, it works, `path` package should be used instead of the `filepath` path package.